src/Campaigns/resources/types.ts

```diff
@@ -21,3 +23,15 @@ declare module "@wordpress/data" {
         dismissNotification(id: string): void
     };
 }
+
+declare global {
```

Comment:
Be careful about declaring global, it makes all files think this is a global variable when its actually only scoped to a specific campaign page.

I like to use [this approach](https://github.com/impress-org/givewp/blob/de80d9b6ca7927e6acd018d7208c2ab1afe6b54c/src/Campaigns/resources/admin/common/getCampaignDetailsWindowData.ts) that you could copy and add to the  alias:
```js
import {getCampaignOptionsWindowData} = '@givewp/campaigns/admin/common'
```